src/pybind/mgr/dashboard_v2/tests/test_nodb.py

```diff
@@ -231,7 +231,7 @@ class NodbModelTest(TestCase):
 
     class SimpleModel(NodbModel):
         a = IntegerField(primary_key=True)
-        b = IntegerField()
+        b = IntegerField(default=None)
```

Comment:
This looks like it's being added because we are no longer using `fields_force_none`, but the we are only checking whether it is `None` in patch line 249. Unfortunately, I don't see the point to this enforcement, nor the single time we check for `None` on previously mentioned line. Can you explain what is it you are trying to achieve?

Reply:
One could think of using a different default than `None`, right.